src/gui/widgets/EffectLabelButton.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * EffectLabelButton.cpp - implementation of class trackLabelButton, a label
+ *                          which is renamable by double-clicking it
+ *
+ * Copyright (c) 2004-2008 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * 
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+
+#include "EffectLabelButton.h"
+
+#include <QMouseEvent>
+#include <QMdiSubWindow>
+
+#include "ConfigManager.h"
+#include "embed.h"
+#include "EffectView.h"
+#include "Effect.h"
+
+namespace lmms::gui
+{
+
+EffectLabelButton::EffectLabelButton(EffectView* _tv, QWidget* _parent) :
+	QPushButton(_parent),
+	m_effectView(_tv),
+	m_iconName()
+{
+	setAttribute(Qt::WA_OpaquePaintEvent, true);
+	setAcceptDrops(false);
+
+	setCursor(QCursor(embed::getIconPixmap("hand"), 3, 3));
```

Comment:
Is this really needed? For other types of buttons, e.g. the "Add effect" button, we do not change the cursor, either. If something is recognizable as a button it shouldn't be necessary to change the cursor IMO.

Perhaps this is something that needs to be decided in general so it's consistent within all of LMMS though.

Interestingly, the cursor changes to a hand if I hover over GitHub's "Add review comment" button in my browser. :sweat_smile:

Reply:
This is done in order to be consistent with the TrackLabelButton class which also changes to a hand cursor on hover. And even just moving around my desktop, it seems like some things activate the hand and some things don't. I'd vote to keep it for consistency with the other `LabelButton` classes.